docs/API-V1/1-INTRODUCTION.md

```diff
@@ -0,0 +1,9 @@
+## API V1
+
+The API v1 of the webapp is callable at /api/v1/{call}?{params}. The response is encoded as JSON object(s).
+
+#### Index
+
+- **SINGLE-PORT**: Information, build history and installation stats related to a particular port.
```

Comment:
These could be relative (clickable) links.

Reply:
Thank you